src/client/app/services/services.js

```diff
@@ -0,0 +1,65 @@
+interview_coding_app.factory('FavStuffService', ['$http', '$q', '$httpParamSerializerJQLike', function ($http, $q, $httpParamSerializerJQLike) {
+        var FavStuffServiceFactory = {};
+        var _getAllFavStuff = function () {
+            var deferred = $q.defer();
```

Comment:
No problem :), It is good to have a feedback of our bad practice in code, It will help me a lot to improve it, thanks.